models/object_detection/model_templates/horizontal-text-detection/tools/visualize_text_detection.py

```diff
@@ -22,7 +22,8 @@
 from pycocotools.cocoeval import COCOeval
 
 from mmdet.datasets import build_dataset
-from mmdet.core.evaluation.coco_utils import results2json
+# TODO(GalyaZalesskaya): fix issue on next line
```

Comment:
@GalyaZalesskaya

Reply:
from mmdet.datasets.coco import result2json (function was moved)

Reply:
@GalyaZalesskaya please fix it, in some PR

Reply:
Is this fix should be merged before your PR? Because I'd like to revisit this script to see if there some new problems besides this one. But it'll take a little time

Reply:
after